pom.xml

```diff
@@ -6,7 +6,7 @@
   <!-- Arifact name and version information -->
   <groupId>net.snowflake</groupId>
   <artifactId>snowflake-ingest-sdk</artifactId>
-  <version>2.0.0</version>
+  <version>2.0.1</version>
```

Comment:
should we also release 2.0.1-unshaded?

Reply:
Yes, that will require a separate PR, at this point, I suggest let's release the shaded one first to make sure there is no other issues